README.md

```diff
@@ -1,2 +1,120 @@
-# mailgun-relayery
-A Go server providing an additional security layer for using the MailGun API
+Mailgun Relayery
+================
+
+Mailgun Relayery is a tool providing an additional security layer for the MailGun API.
+
+**The problem**: MailGun gives out a single API key to each account. For entities dealing with multiple clients or 
+applications, giving out their API key to each of them constitutes a blatant security violation.
```

Comment:
I'd remove "blatant" since it's emotionally loaded -- no need for it in this context?

Reply:
I guess I got carried away there :smile: